lib/Doctrine/DBAL/Schema/AbstractAsset.php

```diff
@@ -48,8 +48,8 @@ protected function _setName(string $name) : void
         }
         if (strpos($name, '.') !== false) {
             $parts            = explode('.', $name);
-            $this->_namespace = $parts[0];
-            $name             = $parts[1];
+            $this->_namespace = array_shift($parts);
+            $name             = implode('.', $parts);
```

Comment:
I think you can achieve the same by using the third argument of `explode`: `explode('.', $name, 2)`

Reply:
https://3v4l.org/95JrL

Reply:
Also, we could have used `list()` here: `[$this->_namespace, $name] = explode('.', $name, 2);`

Reply:
Thanks for the review. I refactored it. What do you think about this version?

Reply:
I think it's better than the first version you proposed 👍